client/src/App.js

```diff
@@ -51,15 +51,47 @@ function App() {
   const groups = useState([
     {
       group_id: 1,
-      name: "Optimistic Yankees",
-      description: "Only yankees allowed.",
-      avatarURL: "https://bit.ly/3cL15Ia",
+      name: "Some Real Eaters",
+      description: "reliable sites for reliable bites",
+      avatarURL: "https://media.techeblog.com/images/cooking-food-cpu.jpg",
+      numReviews: 128,
```

Comment:
We should not need to store num of reviews and members in the groups. These can be figured out by looking at the other states. 

To figure out number of reviews:
- each `review` is linked to a `place_id` and each place is linked to a `group_id`.
- need to access those states and calculate the total number of reviews

To figure out number of members:
- each `user` has `groups` that they are part of. 
- need to access the users and look at their groups to figure out how members each group has.